analyses/interaction-plots/scripts/02-process_mutations.R

```diff
@@ -191,19 +282,21 @@ gene_sample_counts <- maf_filtered %>%
 gene_counts <- gene_sample_counts %>%
   dplyr::filter(sample %in% samples) %>%
   dplyr::group_by(gene) %>%
-  dplyr::summarize(mutant_samples = dplyr::n(), 
-                   total_muts = sum(mutations),
-                   muts_per_sample = mean(mutations)
-                   ) %>%
-  dplyr::arrange(desc(mutant_samples),
-                 desc(muts_per_sample))
+  dplyr::summarize(
+    mutant_samples = dplyr::n(),
+    total_muts = sum(mutations),
+    muts_per_sample = mean(mutations)
+  ) %>%
+  dplyr::arrange(
+    desc(mutant_samples),
+    desc(muts_per_sample)
+  ) %>%
+  dplyr::filter(mutant_samples >= opts$min_mutated |
+    dplyr::row_number() <= 2) # keep at least 2 genes
```

Comment:
Based on this comment, I would expect the logic to be `>= 2` but I might be missing something

Reply:
Or is this to make sure that there are no problems with the call to `coocurrence` ?

Reply:
Yes, this is to prevent errors with coocurrence when there aren't enough genes with the minimal # of mutants (because of few samples or subsetting)